app/graphql/types/query_type.rb

```diff
@@ -1,20 +1,48 @@
 class Types::QueryType < Types::BaseObject
+  field :find_anime, Types::Anime, null: false do
+    description 'Get a single Anime'
+    argument :id, ID, required: false
+    argument :slug, String, required: false
+  end
+
+  def find_anime(id: nil, slug: nil)
+    if id
+      ::Anime.find(id)
+    elsif slug
+      ::Anime.by_slug(slug).first
+    end
+  end
+
   field :anime, Types::Anime.connection_type, null: false do
     description 'Anime in the Kitsu database'
-    argument :slug, [String], required: false
-    argument :id, [String], required: false
   end
 
-  def anime(slug: nil, id: nil)
-    if slug
-      ::Anime.by_slug(slug)
-    elsif id
-      ::Anime.find(id)
-    else
-      ::Anime.all
+  def anime
+    ::Anime.all
+  end
+
+  field :find_manga, Types::Manga, null: false do
+    description 'Get a single Manga'
+    argument :id, ID, required: false
+    argument :slug, String, required: false
+  end
+
+  def find_manga(id: nil, slug: nil)
+    if id
+      ::Manga.find(id)
+    elsif slug
+      ::Manga.by_slug(slug).first
     end
   end
 
+  field :manga, Types::Manga.connection_type, null: false do
+    description 'Manga in the Kitsu database'
+  end
+
+  def manga
+    ::Manga.all
+  end
+
```

Comment:
I can agree with splitting these (1 vs N item queries) but we might wanna outright remove `anime` and `manga` fields for now, until we have actual queries on them

Reply:
I'm querying these for the list view in Wiki that db mods would have access to.